src/Microsoft.VisualStudio.ProjectSystem.Managed/Microsoft.VisualStudio.ProjectSystem.Managed.csproj

```diff
@@ -112,6 +115,7 @@
       <DependentUpon>CompilerCommandLineArgs.xaml</DependentUpon>
     </Compile>
     <Compile Include="ProjectSystem\Configurations\TargetFrameworkProjectConfigurationDimensionProvider.cs" />
+    <Compile Include="ProjectSystem\Utilities\DictionaryEqualityComparer.cs" />
```

Comment:
>ProjectSystem\Utilities\DictionaryEqualityComparer.cs"  [](start = 22, length = 55)

I already included a copy of this class in the unit tests project. You should remove that one and have the code call this one.